apps/taxes/src/modules/app/webhook-response-factory.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiResponse } from "next";
+
+export class WebhookResponseFactory {
```

Comment:
> It also doesn't need to have constructor, you can make a simple static class or object like this

I would like to keep the constructor because I don't want to pass the `res` to each method.

I renamed it to `WebhookResponse`.